### PR TITLE
qualimap: last version with cleaner log and minor bug fixed

### PR DESCRIPTION
**Pull request by lpantano**
There is a better version that avoid to full the log file with the same message. Would be good to have that to update bcbio-nextgen as well. 

The only thing is that bottles need to be updated, but I guess only some people can do it?

thanks

### CODE REVIEW

File: qualimap.rb [hunk 1]
```diff
@@ -12,7 +12,6 @@ class Qualimap < Formula
     sha256 "1916bbaaf3d8e81cb365fa38d55ca45d80e500d917f73b5da8b207286bdf7900" => :mavericks
```
Comment (lpantano) [inline qualimap.rb]: all these hash are wrong, but don't know who has to update https://homebrew.bintray.com/bottles-science

thanks

Comment (iml) [inline qualimap.rb]: Those are updated by @BrewTestBot

**Comment by iml** [comment]
Just a warning for this style of versioning. Putting the day in the beginning can cause an upgrade to not be triggered if the new release is an earlier day in a later month. Homebrew doesn't yet have any rules for versioning but the more popular convention is YYYYMMDD, which increments like an integer.

**Comment by lpantano** [comment]
thanks. Will talk to the author about this, hopefully future version will be better.

**Pull request closed without merging**